lib/utils.bash

```diff
@@ -95,8 +95,7 @@ list_installed_versions() {
   plugin_installs_path="$(asdf_data_dir)/installs/${plugin_name}"
 
   if [ -d "$plugin_installs_path" ]; then
-    for install in "${plugin_installs_path}"/*/; do
-      [[ -e "$install" ]] || break
+    for install in $(ls -1v ${plugin_installs_path}); do
```

Comment:
Is there a way to achieve this without `ls`? I've been considering banning `ls` from the codebase because it's caused a couple bugs in the past.

Reply:
None I can think of or found. Perhaps we can't or shouldn't implement this feature since using `ls` is not universal applicable for all platforms.

Your thoughts? @jthegedus @Stratus3D

Reply:
Would it not be possible to do this with `sort`?

Reply:
I guess part of the problem here is that when we list all versions of a plugin, we call out to the plugin's `list-all` callback, which is responsible for returning versions in the correct order. Here we are dealing with a subset of those same versions, but we don't have a way of calling out to the plugin to sort them.

Reply:
I suppose we could add another plugin callback that would receive a list of versions and would be expected to sort them, but I'm not sure it would be worth the hassle.